jax/_src/nn/functions.py

```diff
@@ -1238,8 +1238,7 @@ def _check_shape_and_dtype(t: Array | None, shape: Sequence[int],
           sliding_window_length=sliding_window,
       )
     case None:
-      # TODO(kaixih@nvidia) Defaults to XLA for now. Will automatically select
-      # best backend.
+      # TODO(kaixih@nvidia) Defaults to XLA for now.
```

Comment:
Could you rephrase the TODO to state what should be done, e.g. "Automatically select the best backend (defaults to XLA for now).", or leave as is?